src/xdk-simple-deploy.ts

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
Can we move this to src/examples/sheets?

Reply:
Moved to the `examples/workbook` folder.